### DISCUSSION TITLE
Does re-running so-setup wipe all data?

### DISCUSSION THREAD

**Discussion by llucasti** [Security-Onion-Solutions/securityonion#10540] [Unsupported Versions]
Hi!

I want to run so-setup again in the vain hope I'll fix an issue.

Will that erase any logs, pcaps, IDS alerts that are already in the database?

**Comment by cm-ops** [comment]
Can you provide a little more about your grid? Is this a standalone or distributed?

**Comment by llucasti** [comment]
It's a standalone install, version 2.3.30.

**Comment by cm-ops** [comment]
2.3.30 is very old, you should consider upgrading to the latest version (2.3.250). Simply re-running so-setup will not wipe all the data, some data will be backed up in backup directories on `/nsm` credentials will be regenerated and you should still see some of the data still within directories in `/nsm`.

What issue are you hoping to fix?

**Comment by llucasti** [comment]
EDIT: I solved the problem by upgrading instead (previously I didn't have Internet access for that SO).
